### ISSUE TITLE
Diffable doesn't seem to work with ST 3.2.2

### ISSUE THREAD

**Issue by bluesweater** [yaroslavyaroslav/Diffable#2]
Hello,

I am on ST 3.2.2 and it doesn't seem to work.

After installation ( console said it installed successfully ) :

**NOTE:** I also installed IncrementalDiff ( ST3 v0.1.0 - https://packagecontrol.io/packages/IncrementalDiff )

**1)**
Sublime text palette > type "Diffable" > nothing happens

**2)**
Right click contextual message on file text area > select "Diffable" > expanded menu items "inline", etc all dimmed (ie. grayed out)

I looked up https://packagecontrol.io/packages/Diffable and it says "ST3" right next to your user name ; yet when I read up your readme file it says Diffable is for ST 4.

Pls clarify. I am not ready to upgrade to ST 4 yet...

Thanks for your hard work but I am sorry to say I am not too happy with this disruption in my workflow (was happy enough with Diffy).

Regardless, have a nice day

**Comment by yaroslavyaroslav** [comment]
Could you please provide exact build number of ST?

Also regarding to your second point it looks like you didn't select two groups in a row, do you?

And the last one, none of additional packages are required, e.g. such package that you  mentioned isn't required.

**Comment by yaroslavyaroslav** [comment]
> I looked up https://packagecontrol.io/packages/Diffable and it says "ST3" right next to your user name ; yet when I read up your readme file it says Diffable is for ST 4

Speaking about that thing, it's a typo in a readme, incremental diff should be available in 3.2.2 as it's written in ST development documentation.

**Comment by bluesweater** [comment]
Hello, thanks for responding so quick.

> Could you please provide exact build number of ST?

3.2.2 build 3211

> Also regarding to your second point it looks like you didn't select two groups in a row, do you?

Not sure what you mean by selecting two groups in a row (I googled but cannot find instruction to do that), but going by your readme instruction of "Open two tabs in a row either way", I have two tabs (ie. two files) open one after another so now one tab is in focus while the other tab is dimmed > command palette > type "Diffable" > nothing ; right click in text area , diffable sub items are all dimmed (grayed out)

Sorry, I may have trouble understanding and you may have to dumb it down for me to explain further how to "select two groups in a row" or "open tabs in a row"... maybe you mean open 1 file after another? If so, that's what I did.

> And the last one, none of additional packages are required, e.g. such package that you mentioned isn't required.
> 
> Speaking about that thing, it's a typo in a readme, incremental diff should be available in 3.2.2 as it's written in ST development documentation.

Understood

I won't be reachable until tomorrow - it's dinner time over here now.

Thanks again.

**Comment by yaroslavyaroslav** [comment]
Thanks, I'll try to get this build and to reproduce this issue locally. 

And just one more question, that plugin that you've mentioned above: IncrementalDiff 0.1.0, is it working on your side?

**Comment by bluesweater** [comment]
> IncrementalDiff 0.1.0, is it working on your side?

Both 
IncrementalDiff 0.1.0 https://packagecontrol.io/packages/IncrementalDiff
and
ST3 internal Incremental Diff http://www.sublimetext.com/docs/incremental_diff.html 
are working
for ST 3.2.2 build 3211 on my end

**Comment by yaroslavyaroslav** [comment]
I was able to reproduce the issue with the build you referenced, and, fortunately, was able to fix it without difficulty. With version 2.0.1, everything should function smoothly in your environment. Thank you for raising up this issue.

Apologies for any inconvenience caused.

**Comment by bluesweater** [comment]
Confirmed v2.0.1 working for ST 3.2.2 build 3211.

Many thanks for looking into this.

**Comment by Kirkman** [comment]
Hi,

I know you closed this issue, but I wanted to let you know I'm having the exact same problem. I'm running Build 4152 on an older Intel Mac Pro, and 4166 on Windows. ST shows that I have installed Diffable, but I can't invoke it via key bindings, the command palette, or contextual menu.

**Comment by yaroslavyaroslav** [comment]
@Kirkman hey, 

A little bit more info required here. Is context menu appears active?

**Comment by Kirkman** [comment]
Okay, some screenshots.

1. The "remove package" screen, which shows Diffable is installed:
![image](https://github.com/yaroslavyaroslav/Diffable/assets/761387/8c50ce8c-5276-4a48-a335-2d8199ff394e)

2. The About box, which shows Build 4166
![image](https://github.com/yaroslavyaroslav/Diffable/assets/761387/0039a019-9a91-43d9-b78e-982434ac0d8b)

3. Screenshot showing the context menu, with two open tabs. Diffable is listed in the context menu, but all options are greyed out.
![image](https://github.com/yaroslavyaroslav/Diffable/assets/761387/7393f4e0-1910-4a83-a754-ad21e310c5e1)

4. Finally, a screenshot showing that no Diffable commands appear in the command palette when I hit [ctrl]-[shift]-[P] and type `diff`.
![image](https://github.com/yaroslavyaroslav/Diffable/assets/761387/c215d025-a438-4c57-a508-805ce3f96791)

**Comment by yaroslavyaroslav** [comment]
@Kirkman as far as I can see, the second tab isn't selected to be active along the first one, while this is required for the diff mode activation.
